ui/Modal/Funding/Link/EthToRadicle.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="typescript">
+  import { Avatar, Button, Emoji, Icon } from "../../../DesignSystem/Primitive";
+  import { Copyable } from "../../../DesignSystem/Component";
+
+  import type { Identity } from "../../../src/identity";
+  import type { Address } from "../../../src/funding/pool";
+
+  export let address: Address;
+  export let identity: Identity;
+
+  export let onCancel: () => void;
+  export let onConfirm: () => void;
+</script>
+
+<style>
+  .wrapper {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  header {
+    padding: 0 var(--content-padding);
+  }
+
+  .from-to {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+
+    width: 100%;
+    height: 10.625rem;
+
+    border: 1px solid var(--color-foreground-level-3);
+    background-color: var(--color-foreground-level-1);
+
+    margin-top: var(--content-padding);
+    padding: calc(var(--content-padding) / 2);
+    border-radius: 1rem;
+  }
+
+  .radicle-user {
+    display: flex;
+    align-items: center;
+  }
+  .submit {
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    margin-top: 1.5rem;
+  }
+</style>
+
+<div class="wrapper">
+  <Emoji emoji="🧦" size="huge" />
+
+  <header>
+    <h1 style="margin-top: 1.5rem;">Add your Ethereum account to Radicle</h1>
+    <p
+      style="margin-top: 1.5rem; padding: 0 var(--content-padding)"
+      class="typo-text">
+      Are you sure you want to add your Ethereum address to your Radicle
+      account?
+    </p>
+  </header>
```

Comment:
Could this be extracted as it is the same for `EnterPassphrase` and `RadicleToEth`? 
This would make editing eg the text confined to one space

Reply:
Good catch!

Reply:
@MeBrei actually, we are unsure whether `EnterPassword` will be used in the end so I'd rather leave it as is and wait another few weeks to get back to this work and improve what we actually end up with.